app/pages/Auth/CreateWallet.view/CreateWallet.test.tsx

```diff
@@ -0,0 +1,44 @@
+// import React from 'react';
+
+// import { act, render } from '@testing-library/react';
+// import userEvent from '@testing-library/user-event';
+
+// import '../../../testUtils/i18nForTests';
+// import { UnlockWalletView } from './CreateWallet.view';
```

Comment:
Instead of commenting all lines, change the file name from ...test.tsx to ...TEST.tsx

Reply:
I will be creating these tests soon, just commented out until I do